src/App.js

```diff
@@ -71,6 +72,13 @@ class App extends Component {
       <Router>
         <StyledApp>
           <Route exact path="/" component={Login} />
+          <PrivateRoute exact path="/dashboard/statistics" component={StatisticsVue} />
```

Comment:
Let's think about the organization of the routes for the future, and how to include necessary global components